db_parser.py

```diff
@@ -42,7 +42,31 @@ def _get_props(keyword, text):
 
     return fields
 
-
+def remove_comments(line):
```

Comment:
Will revise the function to ensure strings with multiple # 's are preserved. This will be implemented in the unit test refactoring as it comes under testing the db parser (see ticket https://github.com/ISISComputingGroup/IBEX/issues/2450 )